tests/features/content_types/content_type_mgmt.feature

```diff
@@ -0,0 +1,49 @@
+@api
```

Comment:
No no, "mgmt" is a common shortcut for "management". I created this file for all scenario related to the content  type life cycle and management.
I except to see more scenarios with other content types; maybe a scenario implying 2 content types...